pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -318,6 +319,13 @@ func (k *Kubernetes) UpdateKubernetesObjects(name string, service kobject.Servic
 		template.Spec.Containers[0].TTY = service.Tty
 		template.Spec.Volumes = volumes
 
+		memoryResourceList := api.ResourceList{
+			api.ResourceMemory: *resource.NewQuantity(
+				int64(service.MemLimit), "RandomStringForFormat")}
+
+		template.Spec.Containers[0].Resources.Limits = memoryResourceList
+		template.Spec.Containers[0].Resources.Requests = memoryResourceList
```

Comment:
If you don't set request it will be same as limit by default.  [source](https://kubernetes.io/docs/user-guide/compute-resources/)
So it probably doesn't matter here.